guides/Implementierungsleitfaden-ISiK-Dokumentenaustausch-401/Einfuehrung/Datenobjekte/Index.page.md

```diff
@@ -9,8 +9,7 @@ Folgende Datenobjekte aus dem Modul [ISiK Basis Stufe 4](https://simplifier.net/
 * [Binary](https://simplifier.net/guide/isik-basis-v4/ImplementationGuide-markdown-Datenobjekte-Datenobjekte_Binary?version=current)
 
 Die Verwendung in diesem Modul bedeutet:
-Wenn dieses Modul auf Datenobjekte aus dem Basismodul referenziert, dann MÜSSEN die referenzierten FHIR-Ressourcen konform zu den Vorgaben aus dem Basismodul sein. 
-Die Referenzierung von im Basismodul spezifizierten Datenobjekten, bedeutet nicht, dass die Interaktionen für die entsprechenden Datenobjekte aus dem Basismodul auch im Modul Dokumentenaustausch bestätigungsrelevant sind.
+Wenn dieses Modul auf Datenobjekte aus dem Basismodul referenziert, dann MÜSSEN die referenzierten FHIR-Ressourcen konform zu den Vorgaben aus dem Basismodul sein.
```

Comment:
issue: in der aktuellen Form ist es für Hersteller nicht ersichtlich welche Vorgaben aus dem Basismodul konkret umzusetzen sind - ALLE oder nur ausgewählte?

suggestion: 

1) konform zu allen Vorgaben an diese Ressourcen aus dem Basismodul sein (Profilkonformität, Interaktionen einschließlich Suchparameter). 

2) konform zu Vorgaben an diese Ressourcen aus dem Basismodul sein (Profilkonformität, Interaktionen einschließlich Suchparameter). Die konkreten Vorgaben zu Interaktionen einschließlich Suchparameter werden noch präzisiert.

Reply:
@simoneOnFhir was sagst du zu den Formulierung 1) und 2) als Ergänzung. Mit 1) wird klargestellt was hier Konformität bedeutet und mit 2) der Status als WIP gekennzeichnet?

Reply:
Wenn, dann nur 1.) da wir derzeit nicht vorhaben, an den Suchparametern irgendwelche Präzisierungen vorzunehmen. Die Anforderungen an die Suchparameter wurden ohnehin kräftig entschäft. Ich denke, nicht, dass es Sinn macht, hier noch weitere Lockerungen vorzunehmen.

1.) finde ich ok und inhaltlich richtig, müsste aber m.E. als Breaking Change hervorgehoben werden, da der Satz das genaue Gegenteil von dem besagt, was vorher dort stand. 
(das war ja das, was wir mit der - zugegeben absichtlich - etwas vagen Formulierung verhindern wollten, bis wir Klarheit darüber haben, wie wir Abhängigkeiten zwischen Modulen künftig handhaben wollen...)

Reply:
@simoneOnFhir , @alexey-tschudnowsky dann nochmal ein Vorschlag :

3 ) Wenn dieses Modul auf Datenobjekte aus dem Basismodul referenziert, dann MÜSSEN die referenzierten FHIR-Ressourcen konform zu Vorgaben an diese Ressourcen aus dem Basismodul sein (Profilkonformität, Interaktionen). Die konkreten Vorgaben zu Interaktionen und Abhängigkeiten zwischen Modulen werden noch präzisiert.

Ok ?

Reply:
👍